protocols/v2/noise-sv2/src/signature_message.rs

```diff
@@ -48,6 +80,12 @@ impl SignatureNoiseMessage {
             true
         }
     }
+
+    // Signs a [`SignatureNoiseMessage`] using the provided keypair (`kp`).
```

Comment:
Is this syntax (`kp`) something agreed on or being used elsewhere? first time I notice it..

Reply:
short form for keypair

Reply:
oh, I mean the syntax with the round brackets ( ` x ` ) is this how we refer to variables?